addons/OPT/tracker/fn_clientInit.sqf

```diff
@@ -0,0 +1,311 @@
+	},
+	[]
+
+] call CFUNC(addEventHandler);
+
+//Event setMarkerText
+[
+	OPT_SET_MARKER_TEXT, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call OFUNC(setMarkerText);
+
+	},
+	[]
+
+] call CFUNC(addEventHandler);
+
+//Event setMarkerIcon
+[
+	OPT_SET_MARKER_ICON, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call OFUNC(setMarkerIcon);
+
+	},
+	[]
+
+] call CFUNC(addEventHandler);
+
+//Event setMarkerColor
+[
+	OPT_SET_MARKER_COLOR, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call OFUNC(setMarkerColor);
+	},
+	[]
+
+] call CFUNC(addEventHandler);
+
+//Event setMarkerHover
+[
+	OPT_SET_MARKER_HOVER, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call OFUNC(setMarkerHover);
+	},
+	[]
+
+] call CFUNC(addEventHandler);
+
+//Event removeMarkerHover
+[
+	OPT_REMOVE_MARKER_HOVER, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call OFUNC(removeMarkerHover);
+	},
+	[]
+
+] call CFUNC(addEventHandler);
+
+//Event removeMarker
+[
+	OPT_REMOVE_MARKER, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call OFUNC(removeMarker);
+	},
+	[]
+
+] call CFUNC(addEventHandler);
+
+//Feststellung ob Dialog vorhanden ist und Eventauslösung 
+[
+	{
+		
+		//OPT Karten-Dialog
+		if (!(isNull ((findDisplay 444001) displayCtrl 10007))) then
+		{
+			[OPT_KARTEN_DIALOG_OPEN, []] call CFUNC(localEvent)
+		};	
+		
+		//BIS Artillery Dialog
+		if (!(isNull ((findDisplay -1) displayCtrl 500))) then
+		{
+			[BIS_ARTILLERY_DIALOG_OPEN, []] call CFUNC(localEvent)
+		};	
+		
+		//BIS UAV Dialog 
+		if (!(isNull ((findDisplay 160) displayCtrl 51))) then
+		{
+			[BIS_UAV_DIALOG_OPEN, []] call CFUNC(localEvent)
+		};		
+	},
+	1, //Zeit verzörgerung für Durchlauf 1/sek
+	[]
+	
+] call CFUNC(addPerFrameHandler);
+
+//Markerzähler einrichten
+GVAR(idMarkerConuter) = 0;
+
+//Markerspeichern einrichten
+GVAR(markerDatenArray) = [];
+
+GVAR(markerIDArray) = [];
+
+//Hilfs Variable für Zeitkontrolle
+GVAR(markerZeitNormalcheck) = 0;
+
+//versetzt den Marker zum Objekt
+
+DFUNC(markerVersetzen) = 
+{
+	params
+					
+	[
+		["_markerID",0],
+		["_trackingobjekt",nil]
+	];
+
+	//Assertions
+	ASSERT_IS_NUMBER(_markerID);
+	ASSERT_IS_OBJECT(_trackingobjekt);
+
+	//Marker Name erstellen
+	private _markerNamen = format ["OPTMarker%1",_markerID];
+
+	//BIS Marker
+	_markerNamen setmarkerpos getPosWorld _trackingobjekt;
+	_markerNamen setmarkerdir getdir _trackingobjekt;
+	
+};
+
+//Kontrolle ob Karte oder Kartendialog geöffnet ist.
+
+DFUNC(kartenCheckOpen) = 
+{
+	
+	private _KartenOpen = false;
+	
+	// OPT Kartendialog 
+	if (!(isNull ((findDisplay 444001) displayCtrl 10007))) then
+	{
+		_KartenOpen = true;	
+	};
+
+	//BIS Artillery Dialog 
+	if (!(isNull ((findDisplay -1) displayCtrl 500))) then
+	{
+		_KartenOpen = true;	
+	};	
+
+	//BIS UAV Dialog
+	if (!(isNull ((findDisplay 160) displayCtrl 51))) then
+	{
+		_KartenOpen = true;	
+	};
+
+	//Mini GPS oder Karte
+	if (visibleMap or visibleGPS) then
+	{
+		_KartenOpen = true;	
+	};	
+
+_KartenOpen
```

Comment:
Die ganzen Boolean-Bedingungen können in ein Statement vereint werden:

`!(isNull ((findDisplay 444001) displayCtrl 10007)) || !(isNull ((findDisplay -1) displayCtrl 500)) || !(isNull ((findDisplay 160) displayCtrl 51)) || visibleMap || visibleGPS`